include/swift/ABI/Task.h

```diff
@@ -327,6 +459,19 @@ class YieldingAsyncContext : public AsyncContext {
   }
 };
 
+/// An asynchronous context within a task that describes a general "Future".
+/// task.
+///
+/// This type matches the ABI of a function `<T> () async throws -> T`, which
+/// is used to describe futures.
```

Comment:
To be precise: it's the signature of the function taken by `runDetached` and `Task.Group.add`.  We basically fully set up the call except for any formal arguments (e.g. closure-context arguments).

Reply:
Sure.